### PR TITLE
Шатров Виктор, 2538

### PR DESCRIPTION
**Pull request by vi34**

### CODE REVIEW

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
-* Created by thevery on 11/09/14.
-*/
 class WhirlView extends SurfaceView implements Runnable {
     int [][] field = null;
-    int width = 0;
-    int height = 0;
-    int scale = 4;
+    int step = 0;
+    final int WIDTH = 240;
+    final int HEIGHT = 320;
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:19]: `public static final`

**Comment by thevery** [comment]
ok = 8
20 FPS

File: app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java [hunk 2]
```diff
@@ -60,54 +76,52 @@ public void run() {
...
-        height = h/scale;
+        screen = new Rect(0,0,w,h);
         initField();
     }
 
     void initField() {
-        field = new int[width][height];
+        paint.setTextSize(50f);
+        paint.setColor(0xFFFFFFFF);
+        field = new int[3][WIDTH * HEIGHT];
```
Comment (thevery) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:86]: why `3`? it seems that you are using only 2.

Comment (vi34) [inline app/src/main/java/ru/ifmo/md/lesson1/WhirlView.java:86]: Third is array of colors used to create bitmap

**Pull request closed without merging**